### ISSUE TITLE
Examples 1 and 2 Not Working

### ISSUE THREAD

**Issue by dinocaputo** [MicrosoftDocs/office-docs-powershell#10424]
Examples 1 and 2 are NOT WORKING. 
While they do not error in PowerShell, there is no subsequent Holiday created in Holidays in TAC.  
Powershell returns the schedule but it cannot be used in TAC or applied to any Auto Attendant

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 93094674-2964-b558-4543-ac3f60ee1e8f
* Version Independent ID: 5b46aa89-0965-e0d2-6e73-e831651e0271
* Content: [New-CsOnlineSchedule (SkypeForBusiness)](https://learn.microsoft.com/en-us/powershell/module/skype/new-csonlineschedule?view=skype-ps)
* Content Source: [skype/skype-ps/skype/New-CsOnlineSchedule.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/main/skype/skype-ps/skype/New-CsOnlineSchedule.md)
* Service: **skypeforbusiness-powershell**
* GitHub Login: @tomkau
* Microsoft Alias: **tomkau**

**Comment by shivtorov** [comment]
Well, the code in examples 1 and 2 defines business hours schedule that we then can use in `New-CsAutoAttendantCallHandlingAssociation` during Auto Attendant creation. See [examples](https://learn.microsoft.com/powershell/module/skype/new-csautoattendantcallhandlingassociation) 

If you created them but have not attached them to Auto Attendants, then they just sitting there doing nothing.

You can find CsOnlineSchedule objects that are not attached to anything:
```powershell
Get-CsOnlineSchedule | where Type -EQ 'WeeklyRecurrence' | where { -not $_.AssociatedConfigurationIds }
```
To delete unused ones:
```powershell
Get-CsOnlineSchedule | where Type -EQ 'WeeklyRecurrence' | where { -not $_.AssociatedConfigurationIds } | Remove-CsOnlineSchedule
```

I guess it would make sense to update description for examples 1 & 2 with some reference to `New-CsAutoAttendantCallHandlingAssociation`

**Comment by dinocaputo** [comment]
Yes agree that makes sense since example 3 does create an actual Holiday perhaps a reference for any business and after hours examples that more configuration is required before you will see anything in TAC.
